packages/react-components/react-skeleton/src/components/SkeletonItem/useSkeletonItemStyles.styles.ts

```diff
@@ -8,7 +8,7 @@ export const skeletonItemClassNames: SlotClassNames<SkeletonItemSlots> = {
   root: 'fui-SkeletonItem',
 };
 
-const skeletonWaveAnimationRTL = {
+const skeletonWaveAnimation = {
```

Comment:
This is intentional, since the `wave` animation is just a transform along the x axis, and the bar is a linear gradient, it will actually appear as if it is moving from rtl if the animation starts at 0% since the middle of the linear gradient is shaded

Reply:
ah makes sense!